system/jlib/jexcept.hpp

```diff
@@ -223,5 +223,9 @@ inline bool isFatal(IError * error) { return isFatal(error->getSeverity()); }
 
 extern jlib_decl IError *createError(WarnErrorCategory category, ErrorSeverity severity, int errNo, const char *msg, const char *filename, int lineno=0, int column=0, int pos=0);
 
+#ifndef _WIN32
+int jlib_decl handleSigPipe();
+#endif
+
```

Comment:
Which would be insignificant...

General policy is that platform-checking #ifdef's should if possible be confined to jlib - i.e. better to have code outside jlib call a function that does nothing in win32 than to have to wrap every call to that function in a #ifndef win32